darts/models/forecasting/torch_forecasting_model.py

```diff
@@ -1429,6 +1419,105 @@ def _load_from_checkpoint(self, file_path, **kwargs):
         pl_module_cls = getattr(sys.modules[self._module_path], self._module_name)
         return pl_module_cls.load_from_checkpoint(file_path, **kwargs)
 
+    def load_weights_from_checkpoint(
+        self,
+        model_name: str = None,
+        work_dir: str = None,
+        file_name: str = None,
+        best: bool = True,
+        strict: bool = True,
```

Comment:
Should we allow `strict=False`? Is there a scenario in which this might be required?